### PR TITLE
Implement Collection of People in Backbone

### PR DESCRIPTION
**Pull request by omidbachari**

### CODE REVIEW

**Comment by omidbachari** [comment]
Ready for review @cpjolicoeur

File: app/assets/javascripts/app/views/people.coffee [hunk 1]
```diff
@@ -0,0 +1,28 @@
+PersonView = Backbone.View.extend {
+  className: 'card'
+  template: _.template "<img class='image' src='<%= imagePath %>'></img>" +
+    "<span class='name'><%= name %></span>" +
+    "<span class='title'><%= title %></span>" +
+    "<span class='bio'><%= bio %></span>"
+  render: ->
```
Comment (cpjolicoeur) [inline app/assets/javascripts/app/views/people.coffee:8]: Prefer leaving a single line of whitespace between function definitions.  Combining class level keys (like `className`, `template`, etc... without gaps are fine)

File: app/assets/javascripts/app/views/people.coffee [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    @$el.html @template(@model.toJSON())
+    this
+}
+
+person = new Person
```
Comment (cpjolicoeur) [inline app/assets/javascripts/app/views/people.coffee]: this var probably isn't needed as it's not used at all?

**Comment by cpjolicoeur** [comment]
@omidbachari just small comments, otherwise :+1:

File: app/assets/javascripts/app/views/people.coffee [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+peopleList = new PeopleList
+
+PeopleListView = Backbone.View.extend {
+  collection: peopleList
+  render: ->
+    @collection.forEach(@addOne, this)
+    this
+  addOne: (person) ->
+    personView = new PersonView {model: person}
```
Comment (omidbachari) [inline app/assets/javascripts/app/views/people.coffee]: here's where that variable is called. looking at it all together, is this a correct usage?

**Pull request merged**